toolchains/docker/toolchain.bzl

```diff
@@ -48,11 +54,18 @@ def _toolchain_configure_impl(repository_ctx):
         Label("@io_bazel_rules_docker//toolchains/docker:BUILD.tpl"),
         {
             "%{DOCKER_TOOL}": "%s" % tool_path,
+            "%{DOCKER_CONFIG}": "%s"% repository_ctx.attr.client_config,
         },
         False,
     )
 
 # Repository rule to generate a docker_toolchain target
 toolchain_configure = repository_rule(
+    attrs = {
+        "client_config": attr.string(
+            default="",
```

Comment:
set default to None

Reply:
Done. Keeping the default on the toolchain (non-repository) rule as we discussed